flink-scala/src/main/scala/org/apache/flink/api/scala/UnfinishedCoGroupOperation.scala

```diff
@@ -61,33 +59,14 @@ class UnfinishedCoGroupOperation[L: ClassTag, R: ClassTag](
 
     // We have to use this hack, for some reason classOf[Array[T]] does not work.
     // Maybe because ObjectArrayTypeInfo does not accept the Scala Array as an array class.
-    val leftArrayType =
+    implicit val leftArrayType =
```

Comment:
These values should be explicitly provided, not implicitly. It makes the code much more understandable